src/rtos/FreeRTOS.c

```diff
@@ -127,6 +129,45 @@ static int nds32_stacking(struct rtos *rtos, const struct rtos_register_stacking
 	return ERROR_OK;
 }
 
+/* take 4 bytes (32 bits) as the default size,
+ * which is suitable for most 32-bit targets and
+ * configuration of configUSE_16_BIT_TICKS = 0. */
+static unsigned freertos_ticktype_size = 4;
```

Comment:
```suggestion
static unsigned int freertos_ticktype_size = 4;
```
OpenOCD style requires this. I realize that this file uses plain `unsigned` all over the place. Some day I'll have to fix that, but for now I'm trying to at least have new code conform to the new style.